moon/config/moon-excavator-round2.json

```diff
@@ -39,15 +38,18 @@
       },
       "rosmsg": {
           "driver": "rosmsg",
-          "in": ["slot_raw", "desired_speed", "tick", "stdout", "request_origin"],
-          "out": ["rot", "acc", "scan", "image", "pose2d", "sim_time_sec", "cmd", "origin"],
+          "in": ["slot_raw", "desired_speed", "tick", "stdout"],
+          "out": ["rot", "acc", "scan", "image", "pose2d", "sim_time_sec", "cmd"],
           "init": {
             "topics": [
               ["/excavator_1/bucket_info", "srcp2_msgs/ExcavatorMsg"],
               ["/excavator_1/joint_states", "sensor_msgs/JointState"],
               ["/qual_2_score", "srcp2_msgs/Qual2ScoringMsg"],
               ["/excavator_1/camera/left/image_raw/compressed", "sensor_msgs/CompressedImage"],
-              ["/excavator_1/camera/right/image_raw/compressed", "sensor_msgs/CompressedImage"]
+              ["/excavator_1/camera/right/image_raw/compressed", "sensor_msgs/CompressedImage"],
+              ["/excavator_1/imu", "sensor_msgs/Imu"],
```

Comment:
I see this as not ready part - is it really necessary for excavator?

Reply:
p.s. now I got it! The problem is that there are two robots and current `rosmsg` does not distinguish which one is which one! I see - sorry for my stupidity!